Resources/Textures/_NF/Clothing/Head/Hats/pirate_hat.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "license": "CC-BY-NC-SA-4.0",
```

Comment:
Licenses are a bit all over the place.  Any reason why?

Reply:
Updating now to fix